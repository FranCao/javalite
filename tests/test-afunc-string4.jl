int main()
{
    string s;
    string t;
    string r;
    s = "Hello World!!";
    t = "FooFooDooDooBar";

    print(s);
    r = substring(s,1,5);
    print(r);
    r = substring(s,4,7);
    prints(r);

    r = substring(t,0,10);
    print(r);
    r = substring(t,4,6);
    print(r);
}