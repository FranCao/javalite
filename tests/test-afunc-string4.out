Hello World!!
ello
o W
FooFooDooDooBar
FooFooDoo
o